auslib/test/admin/views/test_releases.py

```diff
@@ -437,6 +437,10 @@ def testDeleteReadOnlyRelease(self):
         dbo.releases.t.update(values=dict(read_only=True, data_version=2)).where(dbo.releases.name == "d").execute()
         ret = self._delete("/releases/d", username="bill", qs=dict(data_version=2))
         self.assertStatusCode(ret, 403)
+        
+    def testDeleteWithRules(self):
+        ret = self._delete("/releases/d", qs=dict(name='ab', data_version=1))
```

Comment:
Two things here:
1) No need to include "name" in the query string - it's part of the URL.
2) As far as I can tell, "d" is not mapped to. This test is failing because it returns 200! You'll need to update this to choose one of the releases mentioned by the rules from https://github.com/mozilla/balrog/blob/master/auslib/test/admin/views/base.py#L82